Sentiment_Analysis/README.md

```diff
@@ -0,0 +1,29 @@
+#Sentiment Analysis#
+
+
+In Sentiment_Analysis are stored scripts and paper relative to Sentence by Sentence wheat articles sentiment extraction.
+
+Compound sentiment is extracted from sentences of a wheat articles subset using an unmodified version of VADER Analysis Tool. Such
+sentiment scores are then filtered using the Kalman Filter and finally correlation between _Filtered_ _Sentiment_ and daily Wheat Price
+Index is computed. The whole analysis is in the file _"Sentiment_analysis.pdf"_.
+
+###The scripts###
+
+The first script that has to be used is _"sentiment_extraction.py"_.
+
+The script takes wheat articles obtained using _The Reading Machine_, selects a subset of articles where the word _"wheat"_ appears, 
+extracts the sentences containing the word _"wheat"_ from each article, computes sentiment using an unmodified VADER version
+and saves the results into a dataframe called _sentiment_. Such dataframe, which contains Article ID, sentences, compound sentiment and 
+date, is then printed out in a csv file called _"df.csv"_.
+
+_"df.csv"_ is in this folder because 15 observations have been manually cancelled due to format inconsistencies, namely missing article id.
```

Comment:
can't you just add a random id? I mean, it doesn't sound like a show stopper, am I missing something?